common/app/views/support/DropdownMenus.scala

```diff
@@ -0,0 +1,45 @@
+package views.support
+
+import conf.Configuration
+
+
+object DropdownMenus {
+
+  case class DropdownMenuItem (
+    href: Option[String]  = None,
+    dataName: Option[String]  = None,
+    label: String,
+    classList: List[String] = List(),
+    parentClassList: List[String] = List()
+  )
+
+  val accountDropdownMenu: List[DropdownMenuItem] = List(
+    DropdownMenuItem(
+      dataName = Some("nav2 : topbar : comment activity"),
+      label = "Comment activity",
+      classList = List("js-add-comment-activity-link"),
+      parentClassList = List("u-h","js-show-comment-activity")
+    ),
+    DropdownMenuItem(
+      href =Some(s"${Configuration.id.url}/public/edit"),
+      dataName = Some("nav2 : topbar : edit profile"),
+      label = "Edit profile",
+    ),
+    DropdownMenuItem(
+      href =Some(s"${Configuration.id.url}/email-prefs"),
+      dataName = Some("nav2 : topbar : edit profile"),
+      label = "Email preferences"
+    ),
+    DropdownMenuItem(
+      href =Some(s"${Configuration.id.url}/password/change"),
+      dataName = Some("nav2 : topbar : change password"),
+      label = "Change password"
+    ),
+    DropdownMenuItem(
+      href =Some(s"${Configuration.id.url}/signout"),
+      dataName = Some("nav2 : topbar : sign out"),
+      label = "Sign out"
+    )
+  )
+
+}
```

Comment:
Does this file need to live in `/common`?

Reply:
@TBonnin this menu is also used in the [global header](https://github.com/guardian/frontend/pull/18550/files#diff-ddd37e183676780af1a76b20b62b5d2c) (and the mobile sidebar, but i didn't apply it there) too. It's duplicated in a couple places and it felt like a useful reusable bit, plus the file structure can be used for the rest of dropdowns

Reply:
sounds good

Reply:
@TBonnin looking at this with fresh morning eyes i could apply it on the mobile sidebar on the scope of this PR, right now it's a it unclear what the benefits to this file are. Is that okay? didn't want to step on frontend turf too much 😅